utilities/filter_molecule_sets/filter_molecule_sets.py

```diff
@@ -66,123 +57,109 @@ def keep_molecule(mol, max_heavy_atoms = 100,
-            help = "If True keeps molecules that result in warning while loading, default = False, OPTIONAL")
+                      choices = ['True', 'False'],
+                      help = "If True keeps molecules that result in warning while loading, default = False, OPTIONAL")
     parser.add_option('-n', '--heavy', type = 'int', dest = 'heavy', default = 100,
-            help = "Maximum number of heavy atoms per molecule, default = 100, OPTIONAL")
+                      help = "Maximum number of heavy atoms per molecule, default = 100, OPTIONAL")
     parser.add_option('-s', '--SMIRKS', type = 'string', dest = 'smirks_file', default = None,
-            help = "If not None, the file of SMIRKS are parsed and molecules that match that pattern are removed. The file should have a single column with SMIRKS/SMARTS strings where commented lines begin with a '%', default = None, OPTIONAL")
+                      help = "If not None, the file of SMIRKS are parsed and molecules that match that pattern are removed. The file should have a single column with SMIRKS/SMARTS strings where commented lines begin with a '%', default = None, OPTIONAL")
     parser.add_option('-m', '--metals', type = 'int', dest = 'metals', default = 0,
-            help = "Maximum number of metals per molecule, default = 0, OPTIONAL")
+                      help = "Maximum number of metals per molecule, default = 0, OPTIONAL")
     parser.add_option('-H', '--hydrogens', type = 'choice', dest = 'hydrogens', default = 'True',
-            choices = ['True', 'False'],
-            help = "If True, hydrogens are added to the output molecules")
+                      choices = ['True', 'False'],
+                      help = "If True, hydrogens are added to the output molecules")
     parser.add_option('-a', '--atoms', type = 'string', dest = 'atoms', default = None, action = 'store',
-            help = "File name which contains the atomic number of the elements that you do not want to be in your set of molecules.")
+                      help = "File name which contains the atomic number of the elements that you do not want to be in your set of molecules.")
     parser.add_option('-t', '--type', type = 'string', dest = 'atomtype', default = None,
-            help = "Atom type that you do not want in your set of molecules")
+                      help = "List of atom types that you do not want in your set of molecules. Usage: gg,Se (no space between types) ")
     parser.add_option('-y', '--flavor', type = 'choice', dest = 'flavor', default = 'tripos',
-            choices = ['tripos', 'ff'],
-            help = "If you want to include different flavor - options: tripos or ff")
-
-    parser.add_option('-z', '--number', type = 'int', dest = 'number', default = 10000, help = "Limit for the number of molecules in the final set, returns the first z molecules that meet the filtering requirements")
-
-    parser.add_option('-C', '--conformers', type = 'choice', dest = 'conformers',
```

Comment:
The original filter never had this option. @bannanc created this option for personal use, and when we were merging things, this option merged by mistake. If you think it's important, we can add this option.